seacrowd/sea_datasets/muse/muse.py

```diff
@@ -0,0 +1,161 @@
+  booktitle={International Conference on Learning Representations},
+  year={2018}}
+}
+"""
+
+_DATASETNAME = "muse"
+
+_DESCRIPTION = """\
+Contains 110 large-scale ground-truth bilingual dictionaries created and released by Meta using an internal translation tool.
+The dictionaries account for polysemy. The data comprises of a train and test split of 5000 and 1500 unique source words, as well as a larger set of up to 100k pairs.
+It comprises of Europeans languages in every direction, and SEA languages to and from English.
+"""
+
+_HOMEPAGE = "https://github.com/facebookresearch/MUSE#ground-truth-bilingual-dictionaries"
+
+_LANGUAGES = ["fil", "ind", "zlm", "tha", "vie"]
+
+_LICENSE = Licenses.CC_BY_NC_ND_4_0.value
+
+_LOCAL = False
+
+_TRAIN_URL_TEMPLATE = "https://dl.fbaipublicfiles.com/arrival/dictionaries/{src}-{tgt}.0-5000.txt"
+_TEST_URL_TEMPLATE = "https://dl.fbaipublicfiles.com/arrival/dictionaries/{src}-{tgt}.5000-6500.txt"
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+configs = {"tl": ["en"], "id": ["en"], "ms": ["en"], "th": ["en"], "vi": ["en"], "en": ["th", "vi", "tl", "ms", "id"]}
+
+
+class MUSEDataset(datasets.GeneratorBasedBuilder):
+    """Large-scale ground-truth bilingual dictionaries"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = (
+        [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_source",
+                version=SOURCE_VERSION,
+                description=f"{_DATASETNAME} source schema",
+                schema="source",
+                subset_id=f"{_DATASETNAME}_tl_en",
+            ),
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_seacrowd_t2t",
+                version=SEACROWD_VERSION,
+                description=f"{_DATASETNAME} SEACrowd schema",
+                schema="seacrowd_t2t",
+                subset_id=f"{_DATASETNAME}_tl_en",
+            ),
+        ]
+        + [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_{src_lang}_{tgt_lang}_source",
+                version=datasets.Version(_SOURCE_VERSION),
+                description=f"{_DATASETNAME} source schema",
+                schema="source",
+                subset_id=f"{_DATASETNAME}_{src_lang}_{tgt_lang}",
+            )
+            for src_lang in configs
+            for tgt_lang in configs[src_lang]
+        ]
+        + [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_{src_lang}_{tgt_lang}_seacrowd_t2t",
+                version=datasets.Version(_SOURCE_VERSION),
+                description=f"{_DATASETNAME} SEACrowd schema",
+                schema="seacrowd_t2t",
+                subset_id=f"{_DATASETNAME}_{src_lang}_{tgt_lang}",
+            )
+            for src_lang in configs
+            for tgt_lang in configs[src_lang]
```

Comment:
Can we use the 3-letter ISO codes instead of 2-letter codes for the subset names, `text_1_name` values, and `text_2_name` values?